lib/project_types/extension/models/app.rb

```diff
@@ -6,7 +6,7 @@ class App
       include SmartProperties
 
       property! :api_key, accepts: String
-      property! :secret, accepts: String
+      property :secret, accepts: String
```

Comment:
This is a potentially dangerous change, but it's necessary to support flows where the secret isn't required (e.g. CI pushes)